lib/tictactoe/GameRound.ex

```diff
@@ -27,23 +25,23 @@ defmodule Tictactoe.GameRound do
   end
 
   def get_round_state(board, player, currentState) do
-    player1Moves = gameState()[:PLAYER_1_MOVES]
-    player2Moves = gameState()[:PLAYER_2_MOVES]
+    player1Moves = :PLAYER_1_MOVES
+    player2Moves = :PLAYER_2_MOVES
 
     is_win = has_player_won(board, player)
-    is_draw = !Enum.member?(board, space()[:EMPTY])
+    is_draw = !Enum.member?(board, :EMPTY)
     cond do
-      is_draw -> gameState()[:DRAW]
-      is_win && currentState == player1Moves -> gameState()[:PLAYER_1_WON]
-      is_win && currentState == player2Moves -> gameState()[:PLAYER_2_WON]
+      is_win && currentState == player1Moves -> :PLAYER_1_WON
```

Comment:
I wonder if this would read nicer with argument-level pattern matching:

```
def round_state(:win, :player_1_moves), do: :player_1_won
def round_state(:win, :player_2_moves), do: :player_2_won
def round_state(:draw, _), do: :draw
def round_state(_, :player_1_moves), do: :player_2_moves
def round_state(_, :player_2_moves), do: :player_1_moves
```
as this then reads like a truth table.

Also, no need to prefix anything with `get_...` as in Elixir everything is an expression that returns or "gets" a value.

Reply:
Naming I would like to improve together with the big structure refactoring that is coming right after CI/CD.
Was about to ask how to implement those switch-cases properly. Not sure if I get how it should be. Let's do a pair session on this part during structural improvements part 2

Reply:
But it's a good point about `get_` I never thought about it like that:)